Bug: metricsHolder panics when timeoutEvent is happen

```
2023-01-17T14:45:57.619Z    panic    fd    runtime error: invalid memory address or nil pointer dereference
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
    panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa83480]

goroutine 329 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0001d0000, {0x0, 0x0, 0x0})
    go.uber.org/zap@v1.16.0/zapcore/entry.go:234 +0x49b
go.uber.org/zap.(*SugaredLogger).log(0xc0000120d8, 0x4, {0x0?, 0x0?}, {0xc0065bb978?, 0x1?, 0x1?}, {0x0, 0x0, 0x0})
    go.uber.org/zap@v1.16.0/sugar.go:234 +0x13b
go.uber.org/zap.(*SugaredLogger).Panic(...)
    go.uber.org/zap@v1.16.0/sugar.go:123
github.com/ozontech/file.d/logger.Panic(...)
    github.com/ozontech/file.d/logger/logger.go:75
github.com/ozontech/file.d/longpanic.(*LongPanic).recoverUntilTimeout(0xc0000102a0)
    github.com/ozontech/file.d/longpanic/longpanic.go:102 +0x21f
panic({0x16298c0, 0x25fcab0})
    runtime/panic.go:884 +0x212
github.com/ozontech/file.d/pipeline.(*metricsHolder).count(0xc000ff9ca8?, 0xc003800000, 0xc00022acb0?, {0x181b5fe, 0x8}, {0xc0001b89c0?, 0x2654000?, 0xc000ff9cf0?})
    github.com/ozontech/file.d/pipeline/metrics_holder.go:149 +0x280
github.com/ozontech/file.d/pipeline.(*processor).countEvent(...)
    github.com/ozontech/file.d/pipeline/processor.go:261
github.com/ozontech/file.d/pipeline.(*processor).doActions(0xc0001d4420, 0xc003800000)
    github.com/ozontech/file.d/pipeline/processor.go:190 +0xc5
github.com/ozontech/file.d/pipeline.(*processor).processEvent(0xc0001d4420, 0x453d29?)
    github.com/ozontech/file.d/pipeline/processor.go:166 +0x45
github.com/ozontech/file.d/pipeline.(*processor).processSequence(0xc0001d4420, 0x82b?)
    github.com/ozontech/file.d/pipeline/processor.go:145 +0x1e
github.com/ozontech/file.d/pipeline.(*processor).Propagate(0xc0001b89c0?, 0xc00400d000?)
    github.com/ozontech/file.d/pipeline/processor.go:353 +0x25
github.com/ozontech/file.d/plugin/action/join.(*Plugin).flush(0xc003751860)
    github.com/ozontech/file.d/plugin/action/join/join.go:147 +0xdc
github.com/ozontech/file.d/plugin/action/join.(*Plugin).Do(0xc003751860, 0x0?)
    github.com/ozontech/file.d/plugin/action/join/join.go:155 +0x7b
github.com/ozontech/file.d/plugin/action/join_template.(*Plugin).Do(0xc000ff9ed8?, 0xc004634000?)
    github.com/ozontech/file.d/plugin/action/join_template/join_template.go:116 +0x1d
github.com/ozontech/file.d/pipeline.(*processor).doActions(0xc0001d4420, 0xc004634000)
    github.com/ozontech/file.d/pipeline/processor.go:204 +0x17d
github.com/ozontech/file.d/pipeline.(*processor).processEvent(0xc0001d4420, 0x10b54734301?)
    github.com/ozontech/file.d/pipeline/processor.go:166 +0x45
github.com/ozontech/file.d/pipeline.(*processor).processSequence(0xc0001d4420, 0xc004588240?)
    github.com/ozontech/file.d/pipeline/processor.go:145 +0x1e
github.com/ozontech/file.d/pipeline.(*processor).dischargeStream(0xc00022acb0?, 0x100000001?)
    github.com/ozontech/file.d/pipeline/processor.go:136 +0x45
github.com/ozontech/file.d/pipeline.(*processor).process(0xc0001d4420)
    github.com/ozontech/file.d/pipeline/processor.go:124 +0x39
github.com/ozontech/file.d/longpanic.(*LongPanic).Go.func1()
    github.com/ozontech/file.d/longpanic/longpanic.go:69 +0x5b
created by github.com/ozontech/file.d/longpanic.(*LongPanic).Go
    github.com/ozontech/file.d/longpanic/longpanic.go:67 +0x72
```

--- a/pipeline/processor.go
+++ b/pipeline/processor.go
@@ -258,6 +258,9 @@ func (p *processor) tryResetBusy(index int) {
 }
 
 func (p *processor) countEvent(event *Event, actionIndex int, status eventStatus) {
+	if event.IsTimeoutKind() {
+		return
+	}
 	p.metricsValues = p.metricsHolder.count(event, actionIndex, status, p.metricsValues)
 }
 

